### ISSUE TITLE
"Files" Syntax for wildcard

### ISSUE THREAD

**Issue by kohlthaver** [FullHuman/grunt-purgecss#3]
Does the "files" array always convert one source css file to one target file? Or can you give an example on how to use wildcards here (if possible with purgcss, like in the "content" option)

So basically what I want is:
- Purge any css file in given directory structure and output every file (with the same name) to specified output directory

Something like: 

`files: {
          'purged_output/**/*.css': 'scr/css/**/*.css'
        }`

Plan B would be to purge all source css files into one large (purged) css file but I couldn't get that t work, too:

`files: {
          'purged_output/app.css': 'scr/css/*.css'
        }`

results in an empty output file

**Comment by Ffloriel** [comment]
Yes, at the moment `files` takes one _source_ css file and one _destination_ css file.
Being able to specified wildcards for the source files would be an useful feature.
I'll look into it.
